components/list/list-item-generic-layout.js

```diff
@@ -133,18 +147,28 @@ class ListItemGenericLayout extends RtlMixin(LitElement) {
 	}
 
 	render() {
+		const actionClasses = {
+			'd2l-cell': true,
+			'focused': this._actionFocused
+		};
 		return html`
 		<slot name="content-action" class="d2l-cell" data-cell-num="5"></slot>
 		<slot name="outside-control-action" class="d2l-cell" data-cell-num="1"></slot>
 		<slot name="outside-control" class="d2l-cell" data-cell-num="2"></slot>
 		<slot name="control-action" class="d2l-cell" data-cell-num="3"></slot>
 		<slot name="control" class="d2l-cell" data-cell-num="4"></slot>
-		<slot name="actions" class="d2l-cell" data-cell-num="6"></slot>
+		<slot name="actions" class="${classMap(actionClasses)}" data-cell-num="6"
+			@click="${this._clickActionSlot}"
+			@focusout="${this._unclickActionSlot}"></slot>
```

Comment:
This is a `focusout` instead of a blur so we can catch bubbling events

Reply:
This event handler is named as if they are ghost click methods.  Suggest `_onActionSlotClick` and `_onActionSlotFocusOut`.  The `_handle` prefix is fine too.

Reply:
Oh, currently this is wiring up to `click` - does your current sol'n work for keyboard users?  Seems like we'd need to wire-up to `focusin` instead of `click`...?

Note: `d2l-card` had to deal with this as well.  [see here](https://github.com/BrightspaceUI/card/blob/master/d2l-card.js#L252).  It specifically wired up to dropdown open/close events.  Not sure I really have a preference on approach - it specifically targets the dropdown scenario with its fix.

Reply:
I struggled with this a bit - technically I think it's only ever a problem for dropdown open and close events, so I'm fine with specifically targeting those instead of using `focusout` and `close`. Perhaps I should also be testing this with tooltips.

Reply:
I've been switching all the list to `_on` basically, I had `_handleDragHandleblah` and I was like there is too many handles! Also, `_on` is shorter.

Reply:
Changed to `_on`. I've also expanded the fix so that it includes tooltips and is available on every cell using a property called `front-row`. Although it's only these two things that need it right now, you never know!